source/unified-test-format/unified-test-format.rst

```diff
@@ -959,6 +1050,95 @@ The structure of this object is as follows:
   - ``commandName``: Optional string. Test runners MUST assert that the command
     name matches this value.
 
+  - ``serverId``: Defined in `hasServerId`_.
+
+expectedCmapEvent
```

Comment:
Can you create happy-path tests for all the expected events? I see load-balancer tests for these events, but not in valid-pass.

Reply:
Does the `valid-pass/entity-client-cmap-events.yml` file address this?

Reply:
That file doesn't test all the event types listed here.

Reply:
Talked offline. The `entity-client-cmap-events` file tests the simplest ones to ensure drivers are examining CMAP events. I don't want to spend more time on this right now as there are higher priority things to do for the spec tests.